### ISSUE TITLE
[Feature] Improved Accessibility

### ISSUE THREAD

**Issue by anonymous390759** [p2r3/convert#235]
convert.to.it does not comply with various internet accessibility standards.
Here are the following details to improve:

Insufficient Color Contrast [more info](https://dequeuniversity.com/rules/axe/4.4/color-contrast-enhanced?application=axeAPI)

Non-Focusable Elements [more info](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Keyboard?utm_source=devtools&utm_medium=a11y-panel-checks-keyboard&utm_campaign=default#Clickable_elements_must_be_focusable_and_should_have_interactive_semantics)

Invisible Text Labels [more info](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Text_labels_and_names?utm_source=devtools&utm_medium=a11y-panel-checks-text-label&utm_campaign=default#Form_elements_should_have_a_visible_text_label)

These parts of the code should be changed in order to improve user accessibility.

**Comment by p2r3** [comment]
Rather than treating this as a separate issue, I'll tag #73 here to add it for consideration for the redesign. I don't think we'll be changing the frontend much until that comes along.
